packaging/pkg_helpers.bash

```diff
@@ -97,10 +97,19 @@ setup_cuda() {
 #   BUILD_VERSION (e.g., 0.2.0.dev20190807+cpu)
 #
 # Fill BUILD_VERSION if it doesn't exist already with a nightly string
+# Or retrieve it from the version.txt
 # Usage: setup_build_version 0.2.0
 setup_build_version() {
   if [[ -z "$BUILD_VERSION" ]]; then
-    export BUILD_VERSION="$1.dev$(date "+%Y%m%d")$VERSION_SUFFIX"
+    if [[ -z "$1" ]]; then
+      SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+      # version.txt for some reason has `a` character after major.minor.rev
+      # command below yields 0.10.0 from version.txt containing 0.10.0a0
+      _VERSION_BASE=$( cut -f 1 -d a "$SCRIPT_DIR/../version.txt" )
+    else
+      _VERSION_BASE="$1"
```

Comment:
Is there an existing use case for this `else` branch?
If not, it might introduce accidental/unexpected issue.
I think validating `$# -eq 0` is more error prof.